### PR TITLE
convert rocketchat-ui part 2

### PR DESCRIPTION
**Pull request by ggazzo**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-ui/client/lib/sideNav.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		this.initiated = false;
+		this.sideNav = {};
+		this.flexNav = {};
+		this.arrow = {};
+		this.animating = false;
+		this.openQueue = [];
+	}
+
+	toggleArrow(status) {
+		if (status == null) {
```
Comment (rodrigok) [inline packages/rocketchat-ui/client/lib/sideNav.js]: Use default value

File: packages/rocketchat-ui/client/lib/sideNav.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+			}, 50);
+		}
+		return setTimeout(() => {
+			this.animating = false;
+			return typeof callback === 'function' && callback();
+		}, 500);
+	}
+	closeFlex(callback) {
+		let subscription;
+		if (callback == null) {
```
Comment (rodrigok) [inline packages/rocketchat-ui/client/lib/sideNav.js]: Use default value

File: packages/rocketchat-ui/client/lib/cordova/facebook-login.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+				});
+			} else {
+				return fbLoginSuccess(response);
+			}
+		}, function(error) {
+			console.log('getLoginStatus', JSON.stringify(error), error);
+			return callback(error);
+		});
+	}
+	return Facebook.requestCredential(options, credentialRequestCompleteCallback);
```
Comment (ggazzo) [inline packages/rocketchat-ui/client/lib/cordova/facebook-login.js]: there are one return (line 16) if the "if" match this return never happens.

Comment (rodrigok) [inline packages/rocketchat-ui/client/lib/cordova/facebook-login.js]: You are right, sorry :)

**Comment by ggazzo** [comment]
@rodrigok fixed.

**Pull request merged**